.github/dependabot.yml

```diff
@@ -0,0 +1,20 @@
+version: 2
+enable-beta-ecosystems: true
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "daily"
+    groups:
+      dependencies:
+        patterns:
+          - "*"
+  - package-ecosystem: "swift"
```

Comment:
This wouldn't work, we don't commit the Package.resolved file to git, and we shouldn't.

I know the problem, but updating the dependencies for no reason might cause problems for those who intentionally have pinned a package to a lower version.

Reply:
And wouldya look at that, we don't, shouldn't, aaaaaaaaaaand don't have to! That _is_ why https://github.com/vapor-community/swift-dependency-submission has been an action item for me in the SSWG meetings for some time now, after all.